demosplan/DemosPlanCoreBundle/Entity/User/Orga.php

```diff
@@ -99,15 +107,17 @@ class Orga extends SluggedEntity
      * @var DateTime
      *
      * @Gedmo\Timestampable(on="update")
+     *
      * @ORM\Column(name="_o_modified_date", type="datetime", nullable=false)
      */
     protected $modifiedDate;
 
     /**
+     * Comma separated list of cc-Email addresses.
+     *
      * @var string|null
      *
      * @ORM\Column(name="_o_cc_email2", type="string", length=4096, nullable=true)
-     * @Assert\Email(message="email.address.invalid")
      */
```

Comment:
Is there another way to validate multiple email-addresses?
E.g. split them an loop for each?
If so, I would appreciate a follow up Ticket to create this kind of validation of multiple email-addresses.

Reply:
I think the best way would be to refactor this into a to-many relationship to the `EmailAddress` entity and validate the items via `All`.

Reply:
https://yaits.demos-deutschland.de/T31792